vagrant/provisioning/roles/httpd/templates/httpd-ssl.conf

```diff
@@ -305,7 +305,8 @@ ProxyPassReverse /share/res https://{{ alfresco_host }}:7070
 ProxyPass /pentaho https://{{ pentaho_host }}:2002/pentaho
 ProxyPassReverse /pentaho https://{{ pentaho_host }}:2002/pentaho
 
-ProxyPassMatch ^/arkcase/stomp/(.*)/websocket$ wss://{{ arkcase_host }}:8843/arkcase/stomp/$1/websocket
+ProxyPass /arkcase/stomp wss://{{ arkcase_host }}:8843/arkcase/stomp
```

Comment:
This is the fix for the ActiveMQ STOMP issue I had a few months back, the last time I tried to make a Hyper-V VM.  This seems to work fine.  I don't know who came up with the old, weird way.  This way seems to be fine.